examples/src/Examples.js

```diff
@@ -83,10 +86,23 @@ export default class Examples extends Component {
     return null;
   }
 
+  processEvents() {
+    if (this.eventQueue.length > 0) {
+      let event = this.eventQueue[0];
+      this.setState({
+        events: this.eventsExist() ? this.state.events.concat([event]) : [event]
+      }, () => {
+        this.eventQueue.shift();
+        this.processEvents();
+      });
+    }
+  }
+
   onEventAdded(event) {
-    this.setState({
-      events: this.eventsExist() ? this.state.events.concat([event]) : [event]
-    });
+    this.eventQueue.push(event);
+    if(this.eventQueue.length === 1) {
```

Comment:
@jstejada This whole `processEvents()` thing is still a bit of a mystery to me. Why are we processing events only if there is a single item in the queue? I understand that we may get 2 events at the same time (`unfocus + focus`, or `unfocus + select`), but why to process after the first one? Could you please elaborate on this?